frappe_graphql/utils/generate_sdl/doctype.py

```diff
@@ -0,0 +1,174 @@
+import inflect
+
+import frappe
+from frappe.utils import cint
+from frappe.model import default_fields, display_fieldtypes, table_fields
+from frappe.model.meta import Meta
+
+
+def get_doctype_sdl(doctype, ignore_custom_fields=False):
+    meta = frappe.get_meta(doctype)
+    sdl, defined_fieldnames = get_basic_doctype_sdl(meta)
+
+    # Extend Doctype with Custom Fields
+    if not ignore_custom_fields and len(meta.get_custom_fields()):
+        sdl += get_custom_field_sdl(meta, defined_fieldnames)
+
+    # DocTypeSortingInput
+    sdl += get_sorting_input(meta)
+
+    sdl += get_connection_type(meta)
+
+    # Extend QueryType
+    sdl += get_query_type_extension(meta)
```

Comment:
it is better to add another flag to ignore extending query on sdl generation..

Reply:
we wont be able to query those doctypes at all then, right ? 😄

Reply:
if we need to query certain doctypes then running the following to generate the extended query should work right
`bench --site test_site graphql generate_sdl --output-dir <your-app/graphql> --doctype <d1> -dt <d2> -dt <name>`
issues may arise with performance of build schema that's why flag to ignore extended queries can be useful.

Reply:
um, build is done only once, right ? we store it in redis after building and we reuse it. is there any other use for it ?  @assemmarwan what do you think ?

Reply:
@faztp12 built schema is already stored in redis. The end schema size is what affects performance the most see #8 for more details

Reply:
I think it's best that we have control over what to generate so we avoid redundancy or unused queries/mutations. I think by default it will extend queries, otherwise, it will be upto the developer to extend the query

Reply:
so new flag `--ignore-extending-querytype` with description `Toggling this flag will not extend the QueryType with the doctype queries` ?

Reply:
@assemmarwan @e-lobo shall we have this on another PR ?

Reply:
Yes, I think it could be another feature PR 👍🏻